server/app/services/scheduler/filter/port.rb

```diff
@@ -9,14 +9,15 @@ class Port
       # @return [Array<HostNode>]
       def for_service(service, instance_number, nodes)
         candidates = nodes.dup
-        ports = service.ports.map{|p| p['node_port']}
+        ports = service.ports.map { |p| p['node_port'] }
```

Comment:
Not maybe part of this PR, but would it make sense to return early if no ports exposed or `service.net == 'host'`. :)